src/signer.rs

```diff
@@ -6,10 +6,22 @@ use rustls::{
     sign::{Signer, SigningKey},
     Error, OtherError, SignatureAlgorithm, SignatureScheme,
 };
-use sha2::digest::Digest;
+extern crate bcrypt;
+extern crate winapi;
+extern crate windows_sys;
```

Comment:
"extern crate" is not needed in 2021 edition.

Reply:
removed.